### ISSUE TITLE
/swagger endpoint doesn't work

### ISSUE THREAD

**Issue by ttpham0111** [huge-success/sanic-openapi#22]
`/swagger` endpoint causes [index.html](https://github.com/channelcat/sanic-openapi/blob/master/sanic_openapi/ui/index.html) to look for the assets at `/<file_name>` instead of `/swagger/<file_name`. `/swagger/` works because it generates the correct url.

**Comment by kianxineki** [comment]
Thx!

**Comment by Typhon66** [comment]
@ttpham0111 Can you check the latest version 0.3.0 and let me know if this still occurs?

**Comment by endafarrell** [comment]
sanic-openapi ``0.4.0`` on sanic ``0.8.3`` in python ``3.7.1`` works on ``/swagger`` - but not on ``/swagger/`` (due to "<script src="./swagger/swagger-ui-bundle.js"> </script>" pointing to ``/swagger/swagger/swagger-ui-bundle.js`` - note the double "/swagger")

**Comment by hampsterx** [comment]
seems easiest to change readme to mention it should be at "/swagger/" then change

```
blueprint.static('/', dir_path + '/index.html', strict_slashes=True)
```

ie add "strict_slashes"

leave "/swagger" to generate 404

**Comment by chenjr0719** [comment]
> leave "/swagger" to generate 404

Maybe redirect to the right URL would be better than leave it 404, I think.

**Comment by hampsterx** [comment]
yup agreed, code below is bit ugly but works, open to alternatives..

```
@blueprint.route('', strict_slashes=True)
def index(request):
    return redirect("{}/".format(blueprint.url_prefix))

blueprint.static('/', dir_path + '/index.html', strict_slashes=True)
blueprint.static('/', dir_path)
```

**Comment by chenjr0719** [comment]
@hampsterx Would you wanna send a PR for this?

**Comment by hampsterx** [comment]
fixed in 0.5.1
